doc/mgr/dashboard.rst

```diff
@@ -762,6 +762,42 @@ To enable SSO:
 
    ceph dashboard sso enable saml2
 
+.. _dashboard-oauth2-sso-support:
+
+Enabling OAuth2 Single Sign-On (SSO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Ceph Dashboard supports external authentication of users via the
+`OAuth <https://en.wikipedia.org/wiki/OAuth>`_ protocol. You need to
+have :ref:`cephadm` enabled as your orchestrator with an active
+:ref:`deploy-cephadm-mgmt-gateway` and :ref:`deploy-cephadm-oauth2-proxy` services.
+
+From the IDP of choice, Keycloak is the current recomendation and tested solution,
+configure the IDP's client used in the `oauth2-proxy` service configuration to validate the following redirect URLs
+for login_url: `https://<host_name>|<IP_address>/oauth2/callback` and
+the following logout_url: `https://<host_name>|<IP_address>/ /oauth2/sign_out`
+
+Again, from the IDP, we will need a user with a valid role, this user will be the one to perform
+authorization against, we can create a role like: 'administator' to give admin level access to the user.
+
+To disable SSO:
```

Comment:
First enable SSO, then check, then disable.

Reply:
For any particular reason?  @afreen23 

I'm following previous SAML2 command order: 
https://github.com/ceph/ceph/blob/8f415be4a8196c79ac1d613da155bc755fe6d0ae/doc/mgr/dashboard.rst?plain=1#L751

Reply:
Users would want to see enable steps first, then disable. We can correct there too, thanks